pkg/streamingpromql/config.go

```diff
@@ -5,32 +5,37 @@ package streamingpromql
 import (
 	"flag"
 
+	"github.com/grafana/dskit/flagext"
 	"github.com/prometheus/prometheus/promql"
+	"github.com/prometheus/prometheus/promql/parser"
 )
 
 type EngineOpts struct {
-	CommonOpts     promql.EngineOpts
-	FeatureToggles FeatureToggles
+	CommonOpts promql.EngineOpts
+	Features   Features
 
 	// When operating in pedantic mode, we panic if memory consumption is > 0 after Query.Close()
 	// (indicating something was not returned to a pool).
 	Pedantic bool
 }
 
-type FeatureToggles struct {
+type Features struct {
 	EnableAggregationOperations                  bool `yaml:"enable_aggregation_operations" category:"experimental"`
-	EnableVectorVectorBinaryComparisonOperations bool `yaml:"enable_vector_vector_binary_comparison_operations" category:"experimental"`
-	EnableVectorScalarBinaryComparisonOperations bool `yaml:"enable_vector_scalar_binary_comparison_operations" category:"experimental"`
-	EnableScalarScalarBinaryComparisonOperations bool `yaml:"enable_scalar_scalar_binary_comparison_operations" category:"experimental"`
 	EnableBinaryLogicalOperations                bool `yaml:"enable_binary_logical_operations" category:"experimental"`
+	EnableOneToManyAndManyToOneBinaryOperations  bool `yaml:"enable_one_to_many_and_many_to_one_binary_operations" category:"experimental"`
 	EnableScalars                                bool `yaml:"enable_scalars" category:"experimental"`
+	EnableScalarScalarBinaryComparisonOperations bool `yaml:"enable_scalar_scalar_binary_comparison_operations" category:"experimental"`
 	EnableSubqueries                             bool `yaml:"enable_subqueries" category:"experimental"`
-	EnableHistogramQuantileFunction              bool `yaml:"enable_histogram_quantile_function" category:"experimental"`
-	EnableOneToManyAndManyToOneBinaryOperations  bool `yaml:"enable_one_to_many_and_many_to_one_binary_operations" category:"experimental"`
+	EnableVectorScalarBinaryComparisonOperations bool `yaml:"enable_vector_scalar_binary_comparison_operations" category:"experimental"`
+	EnableVectorVectorBinaryComparisonOperations bool `yaml:"enable_vector_vector_binary_comparison_operations" category:"experimental"`
+
+	DisabledAggregations      flagext.StringSliceCSV `yaml:"disabled_aggregations" category:"experimental"`
+	DisabledAggregationsItems []parser.ItemType
+	DisabledFunctions         flagext.StringSliceCSV `yaml:"disabled_functions" category:"experimental"`
 }
 
-// EnableAllFeatures enables all features supported by MQE, including experimental or incomplete features.
-var EnableAllFeatures = FeatureToggles{
+// MQEAllFeatures enables all features supported by MQE, including experimental or incomplete features.
+var MQEAllFeatures = Features{
```

Comment:
Should we leave this as `EnableAllFeatures` now that the struct is called `Features`?